python/README.md

```diff
@@ -100,12 +111,7 @@ To use the C++ implementation, you need to:
 2) Export an environment variable:
 
     $ export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp
-    $ export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2
 
-You need to export this variable before running setup.py script to build and
-install the extension.  You must also set the variable at runtime, otherwise
+You must set this variable at runtime, before running your program, otherwise
```

Comment:
Exactly, the tests set it automatically when --cpp_implementation is specified:
https://github.com/google/protobuf/blob/master/python/setup.py#L162